File: C++/Codes/Oops/Inheritance/basic.cpp
#include <iostream>
using namespace std;

class abs{
    public:
    int a  = 10;
};

class c1 : public abs{
    public:
    int sum(int a, int b){
        return this -> a + b;
    }
};

int main(){
    c1 obj;
    cout << obj.sum(5, 6);
    return 0;
}

File: Java/Basic/Basic.java
package Java.Basic;

import java.util.Scanner;

public class Basic {
       Scanner in = new Scanner(System.in);

        void input() {
              int a, b;
              System.out.print("Enter a no: ");
              a = in.nextInt();
              System.out.print("Enter a no: ");
              b = in.nextInt();
              sum(a, b);
       }

       public void sum(int a, int b) {
              System.out.print("The Sum is : " + (a + b) + "\n");
       }

       public static void main(String[] args) {
              // Basic obj = new Basic();
              // obj.input();
              // float f = 14.5f;
              // System.out.println(f);
              // double e = f; // implicit type casting
              // f = (float)e; // explicit type casting
              // System.out.println(e);
              // System.out.println("Enter a float value: ");
              // f = obj.in.nextFloat();

              // System.out.println(f);
              int arr[][] = new int[8][4];
              System.out.println(Integer.SIZE/8);
              
              



       }
}